src/sbt-test/sbt-header/simple/build.sbt

```diff
@@ -0,0 +1,21 @@
+import de.heikoseeberger.sbtheader.license.Apache2_0
```

Comment:
Your comment was about the check task.

I understand that I have to change 

```
TaskKey[Unit]("check") <<= scalaSource in Compile map { (scalaSource) =>
```

to 

```
val checkFileContents = taskKey[Unit]("Verify file contents match the expected contents")
checkFileContents := {
```

But how do I access the value of `scalaSource` from inside the task definitions without usting the `<<=` operator? I tried it but got the errors above.